syft/pkg/cataloger/dotnet/parse_dotnet_deps.go

```diff
@@ -94,10 +101,16 @@ func parseDotnetDeps(_ context.Context, _ file.Resolver, _ *generic.Environment,
 			reader.Location.WithAnnotation(pkg.EvidenceAnnotationKey, pkg.PrimaryEvidenceAnnotation),
 		)
 
-		if dotnetPkg != nil {
```

Comment:
it looks like this `nil` check got removed, maybe add `if dotnetPkg == nil { continue }`? it's unclear to me if this will cause a panic

Reply:
the answer is: `newDotnetDepsPackage` never returns nil 👍